server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPConfiguration.java

```diff
@@ -164,4 +174,8 @@ public Version getDefaultVersion() {
     public Optional<Long> getMaximumSendSize() {
         return maximumSendSize;
     }
+
+    public Optional<List<String>> getAuthenticationStrategies() {
```

Comment:
IMO this should be moved to JMAPDraftConfiguration and to JMAPRfc6620Configuration.

Here we just do not know to which protocols it applied.

Reply:
> Here we just do not know to which protocols it applied.

Can you explain a little about this? What is diff between `JMAPConfiguration` and `JMAPDraftConfiguration`?

BTW I do not see something like `JMAPRfc6620Configuration`

Reply:
JMAPConfiguration -> parameters for the whole jmap server so draft and rfc-8620

JMAPDraftConfiguration -> only for draft, do not impact RFC-8620 at all

> BTW I do not see something like JMAPRfc6620Configuration

`JmapRfc8621Configuration`